README.md

```diff
@@ -128,6 +128,19 @@ GPU memory.
 Select the version of the TensorFlow library to be used, available
 versions are 1 and 2. Default version is 1.
 
+##### --backend-config=tensorflow,default-max-batch-size=\<int\>
+
+The value to use for max_batch_size during [Autocomplete](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration) when no 
+other value is found. If the `--strict-model-config=false` command-line
+option is used, the tensorflow backend will set the max_batch_size
+of the model to this default value under the following conditions:
+
+1. Autocomplete has determined the model is capable of batching requests. 
+2. max_batch_size is 0 in the model configuration or max_batch_size 
```

Comment:
FYI: The way proto3 is designed, I don't think we can differentiate between the two cases.
The scalar like max_batch_size will be read as 0 even if the user has not provided
max_batch_size.
Still I think for documentation purposes this is fine.

Reply:
Right, I was reading this from a user's perspective. The user has the choice of specifying it or omitting it; the doc says they get the same behavior either way.